sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -297,252 +181,66 @@
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_5</EFDB_Name>
-      <Description>Azimuth_Status_Drive_5</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_6</EFDB_Name>
-      <Description>Azimuth_Status_Drive_6</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_7</EFDB_Name>
-      <Description>Azimuth_Status_Drive_7</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_8</EFDB_Name>
-      <Description>Azimuth_Status_Drive_8</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_9</EFDB_Name>
-      <Description>Azimuth_Status_Drive_9</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_10</EFDB_Name>
-      <Description>Azimuth_Status_Drive_10</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_11</EFDB_Name>
-      <Description>Azimuth_Status_Drive_11</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_12</EFDB_Name>
-      <Description>Azimuth_Status_Drive_12</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_13</EFDB_Name>
-      <Description>Azimuth_Status_Drive_13</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>Azimuth_Status_Drive_14</EFDB_Name>
-      <Description>Azimuth_Status_Drive_14</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <EFDB_Topic>MTMount_cameraCableWrap</EFDB_Topic>
     <item>
-      <EFDB_Name>Azimuth_Status_Drive_15</EFDB_Name>
-      <Description>Azimuth_Status_Drive_15</Description>
-      <Size_in_bytes>1</Size_in_bytes>
-      <IDL_Type>string</IDL_Type>
+      <EFDB_Name>driveState1</EFDB_Name>
```

Comment:
Again, wouldn't an array be better here? Same for angle, speed and current? Perhaps it is not as important here as above where there are many more values.